### PR TITLE
Update kansas.rda and kansas_longer2.dta

### PR DESCRIPTION
**Pull request by sideye**
In `kansas_longer2.dta`, for some reason there were NaNs for North Dakota 1996 only. This PR patches the original stata file and reruns the `clean_data.R` to produce a new `kansas.rda` without these NaN values for better multi-outcome analysis.

### CODE REVIEW

**Comment by sideye** [comment]
FWIW, I ran a compare_df to make sure that the only thing changed between the original and new `kansas.rda` files are the 4 originally NaN rows.

File: tests/testthat/test_multisynth.R [hunk 1]
```diff
@@ -77,7 +77,9 @@ test_that("Separate synth is the same as fitting separate synths", {
                        scm=T, eps_rel=1e-5, eps_abs=1e-5)
     
     # weights are the same-ish
-    expect_equal(c(scm17$weights), c(msyn$weights[-c(15, 16), 2]), tolerance=3e-2)
+    sscm_weights <- unname(c(scm17$weights))
+    mscm_weights <- unname(c(msyn$weights[-c(15, 16), 2]))
+    expect_equal(sscm_weights, mscm_weights, tolerance=3e-2)
```
Comment (ebenmichael) [inline tests/testthat/test_multisynth.R:82]: Could you add a check that the names match up (modulo the treated unit)? That'd make sure nothing funky is happening.

File: R/augsynth.R [hunk 2]
```diff
@@ -52,22 +52,30 @@ single_augsynth <- function(form, unit, time, t_int, data,
...
     augsynth <- fit_augsynth_internal(wide, synth_data, Z, progfunc, 
                                       scm, fixedeff, ...)
-
+    
     # add some extra data
     augsynth$data$time <- data %>% distinct(!!time) %>% pull(!!time)
     augsynth$call <- call_name
     augsynth$t_int <- t_int 
+    
+    if("weights" %in% names(augsynth)) {
```
Comment (ebenmichael) [inline R/augsynth.R:75]: Is there an instance where `augsynth` wouldn't have a `weights` variable?

File: R/multisynth_class.R [hunk 3]
```diff
@@ -86,6 +86,12 @@ multisynth <- function(form, unit, time, data,
                                 time_w = F, lambda_t = 0,
                                 fit_resids = TRUE, eps_abs = eps_abs,
                                 eps_rel = eps_rel, verbose = verbose, long_df = long_df, ...)
+    
+   
+    if("weights" %in% names(msynth)) {
```
Comment (ebenmichael) [inline R/multisynth_class.R:91]: Ditto on the possibility that there's no weights variable. Does this ever happen?

File: vignettes/multi-outcomes-vignette.Rmd [hunk 4]
```diff
@@ -0,0 +1,125 @@
+---
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Multi Outcomes AugSynth Vignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
```
Comment (ebenmichael) [inline vignettes/multi-outcomes-vignette.Rmd:7]: Could you build this vignette to get a markdown file and an html file?? Either `devtools::build_vigenettes()` or just using `knitr::knit()` should be fine.

File: tests/testthat/test_multisynth.R (hunk 1 above)
Comment (sideye) [reply tests/testthat/test_multisynth.R:82]: Added.

File: R/augsynth.R (hunk 2 above)
Comment (sideye) [reply R/augsynth.R:75]: Nope, this is a relic of some debugging I did that I kept. The if statement has been removed.

File: R/multisynth_class.R (hunk 3 above)
Comment (sideye) [reply R/multisynth_class.R:91]: Ditto above.

File: vignettes/multi-outcomes-vignette.Rmd (hunk 4 above)
Comment (sideye) [reply vignettes/multi-outcomes-vignette.Rmd:7]: Added.

**Comment by ebenmichael** [comment]
Looks great, thanks!

**Pull request merged**